30-helm/pingfederate-upgrade/01-background.yaml

```diff
@@ -10,7 +10,7 @@ global:
 
 pingfederate-admin:
   image:
-    tag: 10.2.5-edge
```

Comment:
Any reason to use edge here, vs latest?

Reply:
None that I can see, though the hope is that the user is changing this regardless for their upgrade specification.

Reply:
Perhaps it might be best to force input here via <image tag here> or something similar

Reply:
Yeah, "image tag here" sounds good. We'll just need to mention that those tags need to be replaced in the doc page.